docs/immortal-empires/vampire-counts.md

```diff
@@ -0,0 +1,33 @@
+# Vampire Counts
+
+## Mannfred von Carstein: The Drakenhof Conclave
+
+* **Route I - The Restless Dunes**
+  * The shifting sands of Nehekhara hide many long lost tombs filled with powerful magical artifacts that Mannfred desires. They also hide many foes who would see him undone. They will find only death and an unlife of eternal servitude in these dunes.
+  * Destroy factions:
+    * Khemri, Cult of Sigmar, Chevaliers de Lyonesse
+  * Control provinces:
+    * Shifting Sands, Land of the Dervishes, The Cracked Land, Land of the Dead, Coast of Araby, Land of Assassins, Great Desert of Araby
+  * **Reward**:
+    * _The Ossified Portal_: Creates a dilemma
+    * _The Ossified Monolith_: Campaign movement range: +10% for all armies, Diplomatic relations: +20 with Vampire Counts
+
+* **Route II - Reclaiming the Throne**
+  * The time has come to return to Sylvania and reclaim the von Carstein's ancestral citadel: Castle Drakenhof. In his private chambers in the uppermost tower Mannfred can continue to study the Books of Nagash he has found thus far. But first, Vlad and Isabella must be convinced to recognise his supremacy, or else be staked through the heart and left out for dawn's light to take them.
+  * Destroy factions:
+    * Sylvania
+  * Control province:
+    * Northern Sylvania, Southern Sylvania
+  * Construct landmark:
+    * Malovelent Museum
+  * **Reward**:
+    * _Return of the Count_: Blood Kiss: +3, Weapon Strength: +20 when fighting against The Empire and Kislev
```

Comment:
Weapon Strength: +20 for all units/all armies is broken as fuck. I would go with something milder like +8 Leadership.

Reply:
Nerfed to 5% weap strength and +5 leadership.